index.css

```diff
@@ -0,0 +1,80 @@
+html {
+    font-size: 10px;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+main {
+    border-radius: 0.5rem;
+    margin: 3rem 0.8rem;
+    background-color: #eeeeee;
+    color: #333;
+    line-height: 1.5;
+    text-align: center;
+    font-size: 1.6rem;
+    padding: 1.5rem;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+#title {
+    font-size: 4rem;
+    line-height: 1.2;
+    margin: 2.8rem 0;
+}
+
+#main p {
+    margin-bottom: 1rem;
+}
+
+#img-div {
+    background-color: #ffffff;
+    padding: 0.5rem;
+}
+
+#img {
+    width: 100%;
+    max-width: 100vw;
+    margin: 0 auto;
+    display: block;
+}
+
+#img-caption {
+    margin: 1.5rem 0 0.5rem 0;
```

Comment:
There is a shorter way to implement this style. Please, refactor.  Read about DRY-principle (google it :))